package.json

```diff
@@ -23,8 +23,8 @@
         "css": "node ./scripts/css.js",
         "dev": "node ./scripts/dev.server.js",
         "watchTest": "karma start ./config/karma.config.js",
-        "lint": "prettier --check **/*",
-        "lint-fix": "prettier --write **/* ",
+        "lint": "prettier --check \"./**/*\" --config \".prettierrc.js\" --ignore-path \".prettierignore\"",
```

Comment:
Why not targetting the json instead ?

Reply:
Because I changed it to a .js to be consistent with the other config files

Reply:
Because I changed it to a .js to be consistent with the other config files

Reply:
So where is you change to enable linux ?

Reply:
It's the wrapping of the glob (so `**/*` to `\"./**/*\"`)

Reply:
It's the wrapping of the glob (so `**/*` to `\"./**/*\"`)